packages/gasket-engine/lib/engine.js

```diff
@@ -1,19 +1,18 @@
 const debug = require('debug')('gasket:engine');
-const { Loader, pluginIdentifier } = require('@gasket/resolve');
 
 let dynamicNamingId = 0;
-const isModulePath = /^[/.]|^[a-zA-Z]:\\/;
 
-class PluginEngine {
-  constructor(config, { resolveFrom } = {}) {
-    this.config = config || {};
-    this.loader = new Loader({ resolveFrom });
+class GasketEngine {
```

Comment:
Should we move the JSDoc types and descriptions over to the engine.d.ts file and just import the @type for each of the methods here?

Reply:
Yea, I want to do a significant refactor of types here so let's save for a following PR